master/reference/calicoctl/resources/policy.md

```diff
@@ -46,21 +46,23 @@ spec:
-| egress     | Ordered list of egress rules applied by this policy.                                                                                                                |                 | List of [Rule](#rule) |         |
-| doNotTrack | Indicates that the rules in this policy should be applied before any data plane connection tracking, and that packets allowed by these rules should not be tracked. | true, false     | boolean               | false   |
-
-The `doNotTrack` field is meaningful for [host
-endpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/hostendpoint)
-only.  It does not apply at all to [workload
-endpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint);
-connection tracking is always used for flows to and from those.
-
-[Untracked policy]({{site.baseurl}}/{{page.version}}/getting-started/bare-metal/bare-metal) explains more about how `doNotTrack` can be useful for host endpoints.
+| Field      | Description                                                                                                                                           | Accepted Values | Schema                | Default |
+|------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|-----------------------|---------|
+| order      | (Optional) Indicates priority of this policy, with lower order taking precedence.  No value indicates highest order (lowest precedence)               |                 | float                 |         |
+| selector   | Selects the endpoints to which this policy applies.                                                                                                   |                 | [selector](#selector) | all()   |
+| ingress    | Ordered list of ingress rules applied by policy.                                                                                                      |                 | List of [Rule](#rule) |         |
+| egress     | Ordered list of egress rules applied by this policy.                                                                                                  |                 | List of [Rule](#rule) |         |
+| doNotTrack | Indicates to apply the rules in this policy before any data plane connection tracking, and that packets allowed by these rules should not be tracked. | true, false     | boolean               | false   |
+| preDNAT    | Indicates to apply the rules in this policy before any DNAT.                                                                                          | true, false     | boolean               | false   |
+
+The `doNotTrack` and `preDNAT` fields are meaningful only when applying policy to a
+[host endpoint]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/hostendpoint).
```

Comment:
Couple of thoughts:
- There's a dataplane perf impact for every untracked rule that we add so I think there needs to be a clear requirement for workload untracked policy before we pay the per-packet cost for the dispatch chains.
- It'd be hard to get good semantics for untracked policy for workloads.  Since the raw PREROUTING chain is before the routing table and before NAT, we don't know whether the packets _are_ workload packets at the point the rules are executed.  Also, marking a packet as untracked takes it out of NAT so it breaks things like kube-proxy.